### PR TITLE
#478 Setting to disable AutoRefresh

### PR DESCRIPTION
**Pull request by van800**
Disable refreshing when ctrl-s is pressed without disabling auto-refresh in Unity

### CODE REVIEW

File: resharper/src/resharper-unity/Rider/UnityOptionsPage.cs [hunk 1]
```diff
@@ -27,6 +27,7 @@ public class UnityOptionsPage : OptionsPageBase
             Header("General");
 
             CheckBox((UnitySettings s) => s.InstallUnity3DRiderPlugin, "Install or update Rider plugin automatically");
+            CheckBox((UnitySettings s) => s.AllowAutomaticRefreshInUnity, "Allow Rider to do \"Refresh in Unity\" automatically");
```
Comment (citizenmatt) [inline resharper/src/resharper-unity/Rider/UnityOptionsPage.cs]: "Automatically refresh Unity"?

Comment (van800) [reply resharper/src/resharper-unity/Rider/UnityOptionsPage.cs]: Automatically refresh Assets in Unity?

File: resharper/src/resharper-unity/Settings/UnitySettings.cs [hunk 2]
```diff
@@ -9,6 +9,9 @@ public class UnitySettings
     {
         [SettingsEntry(true, "If this option is enabled, Rider plugin will be automatically installed for Unity projects.")]
         public bool InstallUnity3DRiderPlugin;
+        
+        [SettingsEntry(true, "If this option is enabled, Rider plugin will be automatically call \"Refresh in Unity\".")]
```
[the remark this answers is not in the document]
Comment (citizenmatt) [reply resharper/src/resharper-unity/Settings/UnitySettings.cs]: > If this option is enabled, Rider will automatically notify the Unity editor to refresh assets

and the one above should be changed too, really:

> If this option is enabled, the Rider Unity editor plugin will be automatically installed and updated

File: resharper/src/resharper-unity/Rider/UnityOptionsPage.cs (hunk 1 above)
Comment (citizenmatt) [reply resharper/src/resharper-unity/Rider/UnityOptionsPage.cs]: I think "Automatically refresh assets in Unity" is close enough to auto refresh that Unity people will understand what it does, but includes a bit more context about what gets refreshed.

**Pull request merged**